internal/proxy/handlers.go

```
@@ -9,22 +9,67 @@ import (
 	"github.com/getsentry/sentry-go"
 )
 
+type magicLinkSource interface {
+	Get(ctx context.Context) (string, error)
+}
+
+// how often /api/magic re-reads SSM within one warm Lambda
+// instance.
+// short TTL keeps reviewers on a fresh token without hammering SSM.
+
+const magicCacheTTL = 60 * time.Second
```

golangci-lint (2.12.2)

[error] const magicCacheTTL is unused

(unused)